.github/workflows/drush-security-check.yml

```diff
@@ -0,0 +1,28 @@
+name: Drush Security Check
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  check:
+    name: PHP ${{ matrix.php-versions }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-versions: ['8.1', '8.2']
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup PHP, with composer and extensions
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php-versions }}
+
+      - name: Install Drupal core dependencies and modules
+        run: |
+          composer --no-interaction --no-progress --prefer-dist --optimize-autoloader install
+
+      - name: Run security check
+        run: |
+          ./vendor/bin/drush pm:security
```

Comment:
@vipin-mittal-acquia Does CI fails if there's any security issue ? How do would know, if we are using insecure module ?

Reply:
Yes @vishalkhode1 , the pipeline will fail if any security vulnerabilities are discovered in insecure module.

Reply:
Great, if this is returning non-zero exit fail, if there are security issues, then we are good. 😊

Reply:
@vishalkhode1 : Refer [Screenshot](https://user-images.githubusercontent.com/109503136/211488790-54f3a0c4-28e7-4357-aee4-1f13ed847dde.png), if pipeline fails.